test/python/test_pack.py

```diff
@@ -413,6 +413,31 @@ def test_result_filename_suffix(cartridge_cmd, project_without_dependencies, tmp
         assert expected_filename in os.listdir(tmpdir)
 
 
+@pytest.mark.parametrize('pack_format', ['tgz'])
+def test_invalid_version(cartridge_cmd, project_without_dependencies, tmpdir, pack_format):
+    project = project_without_dependencies
+
+    bad_versions = [
```

Comment:
You don't test "problem" situation when git stash has numbers at start and matched "%d" pattern. I think you could ass something like "123xxx"